src/builtins/commandline.cpp

```diff
@@ -239,6 +241,10 @@ maybe_t<int> builtin_commandline(parser_t &parser, io_streams_t &streams, const
                 paging_mode = true;
                 break;
             }
+            case 'F': {
+                paging_full_mode = true;
```

Comment:
you forgot to update the check for incompatible options
at `src/builtins/commandline.cpp:332`